docs/using/config/configuration.rst

```diff
@@ -1,8 +1,8 @@
 .. _configuration:
 
-===================
-Configuring Flocker
-===================
+===============================
+Specifying your Flocker Cluster
```

Comment:
To put it another way, "Flocker Cluster" is ambiguious - is it the set of servers that Flocker consists of (flocker-control, flocker-dataset-agent, etc.) or the set of servers or datasets the user runs utilizing Flocker. In this case it's the latter, but the title doesn't express that.